frontend/app/src/components/home/groupdetails/Member.svelte

```diff
@@ -62,10 +70,10 @@
             canBlockUser,
     );
 
-    let ownerText = $derived($_("owner"));
-    let adminText = $derived($_("admin"));
-    let moderatorText = $derived($_("moderator"));
-    let memberText = $derived($_("member"));
+    let ownerText = $_(roleAsText(ROLE_OWNER));
```

Comment:
I _think_ these will not change if e.g. the user changes their selected language if this is not in a $derived. Not the end of the world but ...

Reply:
Ah right yeah I'll revert that